go.mod

```diff
@@ -20,44 +20,43 @@ require (
 	github.com/drone/runner-go v1.12.0
 	github.com/drone/signal v1.0.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-chi/chi v1.5.4
 	github.com/go-chi/chi/v5 v5.0.8
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.0
 	github.com/google/wire v0.5.0
-	github.com/harness/lite-engine v0.5.2
-	github.com/hashicorp/nomad/api v0.0.0-20230105155713-bfcb21a550ce
+	github.com/harness/lite-engine v0.5.5
+	github.com/hashicorp/nomad/api v0.0.0-20230321213807-4d31fd323e61
 	github.com/jmoiron/sqlx v1.3.5
-	github.com/joho/godotenv v1.4.0
+	github.com/joho/godotenv v1.5.1
```

Comment:
We faced issue with 1.5 version. It doesn't support `-` in the key that's why we reverted it back.